responses/04_choose-a-template.md

```diff
@@ -0,0 +1,35 @@
+# Choosing a Template
+
+One of the things that makes Learning Lab special is the ability to learn a skill _in context_. As you think about what you want to teach, it's critical to think about the context that you want to teach your topic in. 
+
+Most courses make use of a [:book: template repository](https://lab.github.com/docs/course-ownership-and-repositories#the-repositories). Learning lab clones template repositories for the learner. Template repositories contain starter code or resources to help them find their way.
+
+> The template will need to be under _the same account as the one who owns this course_.
+
+When creating or choosing a template repository for your course, consider two main points.
+
+### Creating relevant content
+
+First, how will this content help teach the concept? For example, if a course is teaching HTML, the success of the finished product should feature HTML.
+
+If you're teaching JavaScript, an app makes sense. If you're teaching Markdown, maybe you don't need a project at all! With CSS, you could use a project relating to design.
+
+Think of a project that is "real" and practical with the skill you're teaching. Don't introduce too many additional, and possibly confusing, concepts.
+
+### Finished product
+
+Second, is the finished product useful? At the end of the course, the finished repository should serve a purpose aside from the lessons. A working game, a portfolio or resume, or a collection of template documents are a few examples.
+
+## Step 4: Reference the template
+
+Select a template repository - or use ours - based on the things above.
+
+### :keyboard: Activity: Edit the config file to reference your chosen template repository
+
+1. Choose a repository to be a template. 
+    - It's important that {{ owner }}, the owner of _this_ repository, is also the owner of the template repository.
+    - It doesn't have to be complete, but it should have at least one commit. If you don't know where to begin, you can [fork this github-games repository](https://github.com/githubtraining/github-games) and use that until you have built one yourself.
+2. [Edit the `config.yml` file in this branch]({{ url }}) to replace the commented `name` and `repo`.
```

Comment:
@brianamarie so I edited those items in the last PR and now I have a conflict. I think we can expect people to follow directions better than I do -- or know how to handle merge conflicts - but we might be able to avoid it if we refactor this step to have them create the PR themselves.

Reply:
Ah - I see. I'm thinking about the steps that we would add by having them create the PR themselves, and the tradeoff of the whole thing. What if we keep the PR created by the bot, but have it created right before it's needed. And, to your point below, before we say we'll merge, we check if it's mergable, and give a few instructions for solving the merge conflict if they do have one? What do you think?

Reply:
Tried this here; f9a2715